### PR TITLE
Flipper IR file data ingestion 🍝🍴😮

### PR DESCRIPTION
**Pull request by zacharesmer**
Add some functions to process flipper IR files into somewhat normalized and less repetitive lists of bits. This will also hopefully be useful for use with any other run length encoded IR data.

### CODE REVIEW

**Comment by zacharesmer** [comment]
I added some code that iterates over all the codes extracted from all the flipper files and sends them to the arduino. Eventually that will make its way into some kind of GUI for tagging codes with different attributes to try and categorize them better.

**Comment by zacharesmer** [comment]
I added some code that iterates over all the codes extracted from all the flipper files and sends them to the arduino. Eventually that will make its way into some kind of GUI for tagging codes with different attributes to try and categorize them better.

File: python_tools/flipper_file_to_ones_and_zeroes.py [hunk 1]
```diff
@@ -1,19 +1,101 @@
...
+            arduino.write(bytes(arduino_string_ver, 'utf-8'))
+            input("Press enter for next")
+            print("Command sent to Arduino.")
+        except ValueError as e:
+            print(e)
 
-for l in flipper_file_to_binary("../raw_wild_ir_captures/cleveland_cavaliers_2022_home_opener_recorded_by_dani/Remote20.ir"):
-    print(l)
\ No newline at end of file
+if __name__ == "__main__":
```
Comment (danielweidman) [inline python_tools/flipper_file_to_ones_and_zeroes.py:100]: I like this pattern of having a way to run the file to do a thing but also just be able to import the functions separately :)

File: python_tools/pixmob_conversion_funcs.py [hunk 2]
```diff
@@ -37,22 +37,26 @@ def bits_to_run_lengths_microseconds(bit_list, pulse_length=cfg.PULSE_LENGTH):
...
 
 def run_lengths_to_bits(run_length_list, pulse_length=cfg.PULSE_LENGTH, acceptable_error=1):
-    # Convert list of run lengths to 1s and 0s based on the pulse length in the kwarg
-    # Resulting lists always start with 1
-    # If acceptable_error is set to something less than 1, an exception will be raised if we find a remainder
-    # greater than that fraction of the pulse length.
-    # Example [700, 2100, 1400] -> [1, 0, 0, 0, 1, 1]
-    # Additional example with error check: If acceptable_error is set to .1, [700, 1900, 1400] would cause an error
-    # because 1900 is 200 off from the closest multiple of 700, and 200 is larger than .1 * 700 = 70
+    """
```
Comment (danielweidman) [inline python_tools/pixmob_conversion_funcs.py:40]: Great description!

**Review by danielweidman — APPROVED** [review]
This is so good! It will make going through these recording data files at least 10 times more efficient. With this (and some time), I bet we'll find a lot of interesting data I missed the first time around!

**Pull request merged**